test/test_autograd.py

```diff
@@ -1231,6 +1243,84 @@ def backward(self, grad_output):
         c.backward(torch.ones(c.size()))
         self.assertEqual(x.grad.data, torch.ones(x.size()))
 
+    def run_conv_double_back_test(self, kern, stride, padding, chan_in, chan_out,
```

Comment:
I just realized this is in `test_autograd.py`, while it really should be in `test_nn.py`